jobs/get_all_cards_by_date_range_gold.py

```diff
@@ -0,0 +1,89 @@
+"""
+This script will fetch all the cards for the given user according to the below criteria:
+- get the date within the given date range 
+- save the data to Delta Lake Gold table
+- This script is scheduled to run twice a week
+"""
+
+import os
+import sys
+from datetime import datetime, timedelta
+import argparse
+from loguru import logger
+from pyspark.sql.functions import col, from_utc_timestamp, to_date
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers import spark_helper as sh, common_functions as cf, trello_functions as tf
```

Comment:
Can you remove common_functions and trello_functions as they are not used here.

Reply:
Removed